tests/network/vmi_networking.go

```diff
@@ -308,44 +306,6 @@ var _ = SIGDescribe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:c
 			}, 15)
 			Expect(err).ToNot(HaveOccurred())
 		})
-
-		It("[test_id:1775]should not request a tun device", func() {
```

Comment:
Well, this test is the only one that checks a VM without any interface being created.
So it makes sense to just change it, not to remove it.

If indeed the `tun` device is not requested, I would say we have a network hotplug limitation (or bug?).
But I fully agree with the notion of not checking this detail in the e2e tests.

One side note though: We should recognize that there are cases where intermediate implementations can be checked and treat those as special exceptions. We are lacking a level of integration tests and therefore may need to result to have some tests checking lower level parts.
To clarify, this one is **not** one of them.

Reply:
@EdDev Are we tracking the possible hotplug problem?

Reply:
Can you please clarify what you mean by this?

Do you mean if we ever try to hotplug an iface for a VM that starts with none?

Reply:
>If indeed the tun device is not requested, I would say we have a network hotplug limitation (or bug?).

I was referring to this. Is this a bug, do have tracker?

Reply:
Aha.. I think we were not particularly interested in the scenario which may cause this.
Should we declare this as a known behavior or a bug, I honestly do not know.

Reply:
> > If indeed the tun device is not requested, I would say we have a network hotplug limitation (or bug?).
> 
> I was referring to this. Is this a bug, do have tracker?

In practice, the basic hotplug support is with the help of migration. I.e. one updates the spec, migrates (manually) and gets into the desired state. In this case, there should be no problem with the `tun` device.

There is also an in-place behavior, if the Multus annotation mutation is acted upon (requires a special addition to the cluster). In this case, the hotplug will probably fail if the original VM has been created with no vNICs.

As the in-place hotplug is not the default (Kubevirt is not responsible to deploy it) and the scenario to start with no vNICs and then hotplug one that requires a `tun` device is very special, I would not do anything about this. However, if anyone is going to encounter this edge case, I would be interested to know about it and probably react.